### ISSUE TITLE
dependency list doesn't use ancestor dependencies option

### ISSUE THREAD

**Issue by grahamdyson** [DevSnicket/eunice#164]
#155 made the previous behaviour of including ancestors in dependency counts an option that isn't the default.
The new option isn't used when listing dependencies, so the lists don't match the counts unless the option is switched, from the default, to on.

```yaml
- id: parent
  items:
    - id: first child
      dependsUpon:
        - second child
        - parent sibling
    - second child
- id: parent sibling
```
![issue 164 default off](https://user-images.githubusercontent.com/8238768/92332002-6e8e9f00-f072-11ea-968e-adab565bf862.png)
![issue 164 on](https://user-images.githubusercontent.com/8238768/92332411-ead6b180-f075-11ea-9252-aab039f42090.png)

**Comment by grahamdyson** [comment]
released in [C# 1.22.1](https://devsnicket.com/eunice/csharp/releases#1.22.1)

**Comment by grahamdyson** [comment]
released in [JavaScript 7.16.1](https://devsnicket.com/eunice/javascript/releases#7.16.1)
